Tests/Clappr_Tests/Classes/Plugin/Playback/AVFoundationPlaybackTests.swift

```diff
@@ -1519,31 +1545,31 @@ class AVFoundationPlaybackTests: QuickSpec {
 
                             playback.play()
                             playback.seek(10)
-                            
+
                             expect(playback.state).to(equal(.stalling))
                         }
                     }
-                    
+
                     context("when paused is called") {
                         it("changes state to paused") {
                             let options = [kSourceUrl: "http://clappr.sample/master.m3u8"]
                             let playback = AVFoundationPlayback(options: options)
-                            
+
                             playback.play()
                             playback.pause()
-                            
+
                             expect(playback.state).to(equal(.paused))
                         }
                     }
-                    
+
                     context("when stop is called") {
                         it("changes state to idle") {
                             let options = [kSourceUrl: "http://clappr.sample/master.m3u8"]
                             let playback = AVFoundationPlayback(options: options)
 
                             playback.play()
                             playback.stop()
-                            
```

Comment:
Can you check about all this changes in this file? It was an automatic Lint?

Reply:
These spaces have been removed after a cut and paste in Xcode. Xcode was set to remove unnecessary spaces in the code.

As this file has more changes than spaces, I find it unnecessary to go back as it was, to simply change the code and commit again